docs/api-reference-1/frontity-hooks.md

```diff
@@ -9,7 +9,7 @@ This package is a collection of React hooks that have proven to be pretty useful
 In order to use it, you just have to import the hook you want to use in your theme from `@frontity/hooks/` and place it wherever needed. For example, if we want to use the `useInView` hook:
 
 ```javascript
-import useInView from "@frontity/hooks/useInView";
+import {useInView} from "@frontity/hooks";
```

Comment:
Oops! @juanmaguitar, both `import` sentences are not equivalent. The one you used requires a `index.ts` file in `@frontity/hooks` that exports `useInView` and we don't have it. You need to import the hook directly from its file.